meteor/sofie-server/data-access/repositories/test/mongo-rundown-playlist-repository.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { instance, mock, when } from 'ts-mockito'
+import { Rundown, RundownInterface } from '../../../model/entities/rundown'
+import { MongoEntityConverter } from '../mongo/mongo-entity-converter'
+import { MongoRundownPlaylistRepository } from '../mongo/mongo-rundown-playlist-repository'
+import { RundownRepository } from '../interfaces/rundown-repository'
+import { MongoDatabase } from '../mongo/mongo-database'
+// eslint-disable-next-line node/no-unpublished-import
+import { MongoMemoryServer } from 'mongodb-memory-server'
+import { MongoClient } from 'mongodb'
+
+describe('mongo-rundown-playlist-repository.spec.ts', () => {
+	let mongoServer: MongoMemoryServer
+	let client: MongoClient
+
+	beforeEach(async () => {
+		mongoServer = await MongoMemoryServer.create()
+		client = await MongoClient.connect(mongoServer.getUri())
+	})
+
+	afterEach(async () => {
+		if (client) {
+			await client.close()
+		}
+		if (mongoServer) {
+			await mongoServer.stop()
+		}
+	})
+
+	describe('getRundown', () => {
+		it('has an activationId, return an active rundown', async () => {
+			const rundownId: string = 'activatedRundownId'
+			const testee: MongoRundownPlaylistRepository = await createTesteeForRundown(true, rundownId)
+			const result: Rundown = await testee.getRundown(rundownId)
+
+			expect(result.getActiveStatus()).toBe(true)
+		})
+		it('does not have an activationId, return deactivated rundown', async () => {
```

Comment:
"return _inactive_ rundown" would make more sense from a non techincal standpoint?

Reply:
Changed to the suggestion!